lib/tds.js

```diff
@@ -47,6 +61,7 @@ const MAXIMUM_REGEX_RULES = 900
 // declarativeNetRequest rule to aid the creation of the trackerDomainByRuleId
 // lookup.
 const trackerDomainSymbol = Symbol('trackerDomain')
+const inverseCustomActionSymbol = Symbol('customAction')
```

Comment:
Nit: `customAction` ?

Also, could you expand on the above comment to briefly mention about custom actions?